docs/howto/connection/index.rst

```diff
@@ -88,6 +88,31 @@ Alternatively you may specify each parameter individually:
         --conn-schema 'schema' \
         ...
 
+.. _connection/export:
+
+Exporting Connections from the CLI
+----------------------------------
+
+You may export connections from the database using the CLI. The supported formats are ``json``, ``yaml`` and ``env``.
```

Comment:
Can you describe exactly what each of these file formats looks like? It is very important that the user can look into the documentation to know what such a file looks like. Here is an example of a similar description so you can copy common parts of your docs.
https://airflow.readthedocs.io/en/latest/howto/use-alternative-secrets-backend.html#local-filesystem-secrets-backend
At this point, it makes sense to add a reference to your section.
>  You can create the file by exporting the connections from the database, for more information see: "Exporting connection from the CLI"

Documentation is often the more difficult part than writing an implementation, but I try to fill many gaps in the documentation.

Reply:
@mik-laj Please have a look now. I've added the examples.